openmp/cmake/OpenMPTesting.cmake

```diff
@@ -55,6 +55,9 @@ if (${OPENMP_STANDALONE_BUILD})
   if (MSVC OR XCODE)
     set(DEFAULT_LIT_ARGS "${DEFAULT_LIT_ARGS} --no-progress-bar")
   endif()
+  if (${CMAKE_SYSTEM_NAME} MATCHES "AIX")
```

Comment:
If we want to impose time limit for the lit tests, should the limit also apply to all platform?

Reply:
The time limit depends on the platform so I'd refrain from setting it for other platforms.